### DISCUSSION TITLE
Dudas con respecto a la entrega de las katas

### DISCUSSION THREAD

**Discussion by Tavolordk** [LaunchX-InnovaccionVirtual/CursoIntroPython#138] [Q&A] [answered]
Tengo entendido que vamos a hacer los ejercicios ya sea por medio de hacer un fork, descargar el proyecto, clonarlo o simplemente hacer uno igual nosotros, lo que no entiendo es cómo van a tener la evidencia, haremos un pull request con nuestro nombre incluido o algun lugar donde debemos subir nuestra evidencia en teams? espero me resuelvan esa duda gracias.

**Comment by AzaelBarbosa** [comment]
Buen dia companer, los que entregaras sera el archivo .ipynb ya sea que quieras entrega el que viene en el Repositorio o quieras crear uno tu desde 0, para la entrega, el dia de hoy en la transmision, se nos dara un link con un Formulario donde pondremos nuestro nombre y el link del repositorio donde estarán tus archivos.

**Reply by Tavolordk** [reply]
muchisimas gracias por tu respuesta compañero, me sirvió mucho.

**Comment by DanielOe21** [comment, marked as answer]
Hola @Tavolordk , en los lives que hemos tenido, la instructora nos ha repetido en diversas ocasiones que la entrega de las katas será de la siguiente manera:

- Creas un repositorio en tu cuenta de Github 
- Cargas las katas 
- Copias el link del repositorio
- Registras ese link en un formulario que nos dirán en la siguiente transmisión

Habrá más detalles en la transmisión correspondiente, no te la pierdas! o ve la grabación.

PD: No es necesario hacer un Fork del repo original, ni un clone ni nada de eso, si lo quieres hacer pues es una buena opción, pero el chiste es que tengas en **TU cuenta de Github** un repo con los ejercicios resueltos.

PD2: Si consideras que tu respuesta ha sido contestada, por favor márcala como tal.

PD3: Recuerda que las sesiones anteriores se quedan grabadas en el mismo link, quizá no las hayas visto a detalle por cualquier motivo, pero puedes revisarlas sin problema para que no te pierdas de las instrucciones especiales.
